src/image/image.cpp

```diff
@@ -134,6 +140,8 @@ bool loadPng(const char *filename, bimg::ImageMip &mip, bimg::TextureFormat::Enu
 
     data = (uint8_t*)driver_calloc(datasize, sizeof(uint8_t));
 
+    ffnx_info("%s: data_size=%d width=%d height=%d bit_depth=%d color_type=%X\n", __func__, datasize, _width, _height, bit_depth, color_type);
```

Comment:
Do we want to log this every time or only when `trace_loaders` is enabled?

Reply:
We do not want this log at all I guess, do you want it?

Reply:
Not sure, if it helps debugging why not. Does it help you in case for future crashes to be reported?

Reply:
Okay then, I re-add it

Reply:
Maybe consider marking it as `ffnx_trace` and put it with the usual `trace_all || trace_loaders` if. That would be more than enough

Reply:
Okay sorry I marked resolved before reading your message, it should be good now